qiita_ware/test/test_ebi.py

```diff
@@ -37,6 +43,18 @@ def setUp(self):
         self.sample3_fp = ebi_test_file('sample3.fastq.gz')
 
         self.temp_dir = gettempdir()
+        self.files_to_remove = []
+        self.ppd_to_remove = []
+
+    def tearDown(self):
+        for f in self.files_to_remove:
+            if isdir(f):
+                rmtree(f)
+            else:
+                remove(f)
+
+        for _id in self.ppd_to_remove:
```

Comment:
Good point, made the change but note that I divided the test in readonly and write/read cause without that decorator the test took 0.246s vs. with all the tests with the decorator it took 31.447s so with that split it takes 1.194s.